templates/financial-services/index.html

```diff
@@ -234,11 +234,58 @@ <h2>Reduce IT operations cost</h2>
       }}
     </div>
   </div>
+</section>
 
-  <div class="u-fixed-width">
-    <hr class="p-separator"/>
+<section class="p-strip--suru">
+  <div class="row">
+    <div class="col-6">
+      <div class="p-heading-icon--muted">
+        <div class="p-heading-icon__header">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/fc781c43-Case+study+-+white.svg",
+            alt="",
+            width="32",
+            height="32",
+            hi_def=True,
+            attrs={"class":"p-heading-icon__img p-heading-icon__img--small"},
+            loading="auto|lazy"
+            ) | safe
+          }}
+          <p class="p-heading-icon__title p-heading--4" style=" color: #FFF; line-height: 1.5rem;">Case study</p>
```

Comment:
Can you file an issue on Vanilla for this pattern if its commonly used please

Reply:
Will do